tests/test_db.rs

```diff
@@ -251,3 +251,88 @@ fn test_sequence_number() {
         assert_eq!(db.latest_sequence_number(), 1);
     }
 }
+
+struct OperationCounts {
+    puts: usize,
+    deletes: usize,
+}
+
+impl rocksdb::WriteBatchIterator for OperationCounts {
+    fn put(&mut self, _key: Box<[u8]>, _value: Box<[u8]>) {
+        self.puts += 1;
+    }
+    fn delete(&mut self, _key: Box<[u8]>) {
+        self.deletes += 1;
+    }
+}
+
+#[test]
+fn test_get_updates_since() {
```

Comment:
nit: I'd split this test into 5 smaller tests, just for readability. Also I think you can move it to the separate file into `tests` directory - I'm literally dying because of thousands of lines in this crate.

Reply:
Yeah, I was copying other functions in that test crate, but I agree that splitting is good.